lib/stormpath-sdk/client.rb

```diff
@@ -20,18 +20,18 @@ class Client
     include Stormpath::Util::Assert
     include Stormpath::Resource::Associations
 
-    attr_reader :data_store, :application
+    attr_reader :data_store, :application, :api_key
```

Comment:
Hi ecrisostomo, if I make it protected I wouldnt’t be able to access the api_key in the create_id_site_url method on application class. Also since the end user here is the programmer implementing the gem into his application he would already need to have an api_key, so in my opinion leaving it public doesn’t produce any security issues. What do you think? Also if you disagree, do you have a proposition how to access the api_key from the application class?